Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-09-21
Initial Package Version: 3.0.20
Upstream Status:         Not submitted (it is not an upstream Samba issue)
Origin:                  Randy McMurchy
Description:             Fixes an issue if you have Heimdal installed as
                         Heimdal uses reserved C++ words and the Samba headers
                         now look for these conflicts. This patch comments out
                         the check for C++ reserved words. Heimdal is aware of
                         the issue

$LastChangedBy: randy $
$Date: 2005/09/29 18:25:12 $


--- samba-3.0.20-orig/source/include/includes.h	2005-07-28 13:19:49.000000000 +0000
+++ samba-3.0.20/source/include/includes.h	2005-09-04 13:45:01.000000000 +0000
@@ -25,6 +25,7 @@
 #include "config.h"
 #endif
 
+/*
 #ifndef __cplusplus
 #define class #error DONT_USE_CPLUSPLUS_RESERVED_NAMES
 #define private #error DONT_USE_CPLUSPLUS_RESERVED_NAMES
@@ -36,6 +37,7 @@
 #define delete #error DONT_USE_CPLUSPLUS_RESERVED_NAMES
 #define friend #error DONT_USE_CPLUSPLUS_RESERVED_NAMES
 #endif
+*/
 
 #include "local.h"
 
